docs/integration/ibm-cloud-kubernetes-logs.md

```diff
@@ -0,0 +1,253 @@
+
+The Logagent [input-filter for Containerd](../logagent/input-filter-containerd/) supports:
+
+* Tailing log files from `/var/log/containers/`, `/var/log/pods` and `/var/data/kubeletlogs`
+* Enrichment of logs with podName, namespace, containerName, containerId
+* Joining long log lines over 4KB into one log line
+* Parsing containerd log headers (timestamp, stream, flags)
+* Parsing message content with Logagent's parser library
+
+
+## Default Setup of IBM Cloud Kubernetes Logs
+
+With the default setup you edit env vars to change the configuration.
+
+### Forward all Container Logs
+
+Run Logagent as [Kubernetes DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset).
+
+First, create the [ibm-cloud-logagent-ds.yml](https://github.com/sematext/logagent-js/blob/master/kubernetes/ibm-cloud-logagent-ds.yml) DaemonSet file.
+
+```bash
+curl -o ibm-cloud-logagent-ds.yml  https://raw.githubusercontent.com/sematext/logagent-js/master/kubernetes/ibm-cloud-logagent-ds.yml
+```
+
+Set your `LOGS_TOKEN` in the `spec.env` section in the `ibm-cloud-logagent-ds.yml` file.
+
+Then run the DaemonSet:
+
+```
+kubectl create -f ibm-cloud-logagent-ds.yml
+```
+
+### Use `LOG_GLOB` to Filter Which Container Logs to Forward
+
+Log globs make it easy to use wildcards to filter in/out which log files to tail.
+In the `spec.env.LOG_GLOB` env var you can set values to not include logs from certain containers.
+To read more about log globs, [check this out](https://www.npmjs.com/package/glob).
+
+Here's how you can exclude all logs from the `kube-system` namespace:
+```yaml
+env:
+  - name: LOG_GLOB
+    value: "/var/log/containers/!(*kube-system*.log);/var/log/*.log"
+```
+
+Or, only include logs from the `default` namespace:
+```yaml
+env:
+  - name: LOG_GLOB
+    value: "/var/log/containers/*default*.log;/var/log/*.log"
+```
+
+This is a quick way of including/excluding logs from containers.
+
+## Advanced Setup of IBM Cloud Kubernetes Logs
+
+With the advanced setup you add a `logagent.conf` file as a `ConfigMap` to change the configuration.
+With this config file you have more control over the settings, including:
+
+- dropping logs
+- log routing
+- more fine tuned filtering
+- ability to add audit logs
+
+### 1. Create a `logagent.conf` file
+
+The logagent.conf is the main config file for Logagent.
+
+```yaml
+# logagent.conf
+options:
+  debug: false
+
+input: 
+  files: 
+    - /var/log/*.log
+    - /var/log/containers/*.log
+
+inputFilter:
+  - module: input-filter-k8s-containerd
+
+output:
+  elasticsearch:
+    module: elasticsearch
+    url: https://logsene-receiver.sematext.com
+    index: YOUR_SEMATEXT_LOGS_TOKEN
+```
+
+This particular config above will work the same as using the default setup with env vars.
```

Comment:
Maybe we can add: You may need to adjust the URL to the Sematext location you are using.

Reply:
Added: 

You may need to adjust the `url` to the Sematext region you are using.

Reply:
Also added both urls in the config but one of them is commented out.